precompile/modules/initia_stdlib/sources/table.move

```diff
@@ -224,6 +224,48 @@ module initia_std::table {
         (key, &mut box.val)
     }
 
+    public inline fun get_last_key_and_value<K: copy + drop, V>(table: &Table<K,V>): (K,&V) {
+        let iter = iter(
+            table,
+            option::none(),
+            option::none(),
+            2
+        );
+        if(!prepare<K, V>(&mut iter)) {
+            abort(error::invalid_argument(ENOT_FOUND))
+        };
+        let(key, value) = next<K,V>(&mut iter);
+
+        (key, value)
+    }
+
+    public inline fun loop_table_mut<K: copy + drop, V>(mut_table: &mut Table<K, V>, f: |K, &mut V|) {
+        let iter = iter_mut(
+            mut_table,
+            option::none(),
+            option::none(),
+            1
+        );
+        loop {
+            if (!prepare_mut<K, V>(&mut iter)) { break };
+            let (key, value) = next_mut<K, V>(&mut iter);
+            f(key, value)
```

Comment:
good to add get (stop bool) and break the loop when the callback function return true